components/bl-stripe-payment-component/README.md

```diff
@@ -0,0 +1,71 @@
+# Stripe Payment
+
+Stripe Payment is a component of Backendless UI-Builder designer. You will have a customizable interface which enables
+secure processing of credit card transactions using Stripe.
+
+Customizations include adjustments of amount, minimal amount and currency label. The payment form is configurable, and
+thus you can set visibility of separate fields. Also, you have an option to make the amount field editable by the user.
+
+The component is based on external [stripe-js](https://github.com/stripe/stripe-js)
+and [react-stripe-js](https://github.com/stripe/react-stripe-js) libraries.
+
+## Usage
+
+Before adding the component to the page, open the Marketplace screen and install
+the [Stripe Integration Plugin](https://backendless.com/features/marketplace/stripe-integration-plugin/) from the
+Backendless marketplace. The plugin product is available under the All Services menu.
+
+Then add the component to the page and specify the Publishable key in UI-Builder Settings.
+
+### Styles and Settings
```

Comment:
please change the readme according to the new standard https://github.com/Backendless/ui-builder-library/blob/main/components/bl-leafletMap-component/README.md